oktapam/constants/typed_strings/key_algos.go

```diff
@@ -0,0 +1,41 @@
+package typed_strings
+
+import (
+	"fmt"
+	"strings"
+)
+
+type SSHCertificateType string
+
+const (
+	CertTypeRsa      = "CERT_TYPE_RSA_01"
+	CertTypeRsa256   = "CERT_TYPE_RSA_SHA2_256_01"
```

Comment:
The API docs don't list variants of RSA sig algorithm supported beyond plain CERT_TYPE_RSA_01. See https://developer.okta.com/docs/reference/api/asa/projects/#request-body-2. Are our API docs inaccurate and don't reflect what we support or are these algorithms added here not supported by our backend?

Reply:
So it looks like they are options we [support](https://github.com/okta/platform/blob/master/backend/models/project.go#L57) but not in the [UI](https://github.com/okta/platform/pull/5792) -- should they be added to the UI?

Reply:
It goes beyond UI though because our API docs also don't show those other RSA sig options. I'm inclined to leave it as is because a) if we show them through UI we need to update API docs and b) we are in general trying to deprecate use of RSA (even though those other options are more secure versions of RSA sig alg). It should be a separate feature request (unrelated to TF provider) to 'productize' those other options.